plugins/csp-lod-bodies/src/TileSourceWebMapService.cpp

```diff
@@ -309,35 +281,75 @@ bool TileSourceWebMapService::getXY(int level, glm::int64 patchIdx, int& x, int&
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-std::optional<std::string> TileSourceWebMapService::loadData(int level, int x, int y) {
+std::optional<std::string> TileSourceWebMapService::loadData(
+    glm::int64 patchIdx, int level, int x, int y) {
```

Comment:
We should stick to std types if available.

```suggestion
    int64_t patchIdx, int level, int x, int y) {
```